conjure-java/build.gradle

```diff
@@ -18,6 +18,16 @@ apply plugin: 'com.palantir.external-publish-application-dist'
 
 mainClassName = 'com.palantir.conjure.java.cli.ConjureJavaCli'
 
+application {
+    applicationDefaultJvmArgs.addAll([
```

Comment:
The API for this is `Iterable<String> getApplicationDefaultJvmArgs();`. This code is relying on the fact the default value is an `ArrayList` which is a bit risky.

Reply:
But fine I guess as nothing else is adding to it here.